wpimath/src/main/java/edu/wpi/first/math/geometry/Translation3d.java

```diff
@@ -126,6 +139,15 @@ public double getZ() {
     return m_z;
   }
 
+  /**
+   * Returns a vector representation of this translation.
+   *
+   * @return A Vector.
+   */
+  public Vector<N3> getVector() {
```

Comment:
Name this `toVector()` to match `toTranslation2d()`. Also needs a C++ version (e.g., `Eigen::Vector3d Translation3d::ToVector()`).